package.json

```diff
@@ -31,7 +31,7 @@
     "estree-walker": "2.0.2",
     "glob": "^10.4.5",
     "graceful-fs": "^4.2.9",
-    "node-gyp-build": "^4.2.2",
+    "node-gyp-build": "^4.8.4",
```

Comment:
I think this should be a separate PR since its not a `chore:`, its a `fix:`

(changes to `dependencies` affect consumers)

Reply:
Reverted